src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -153,6 +153,10 @@ public function load(array $configs, ContainerBuilder $container)
             $definition->replaceArgument(2, E_COMPILE_ERROR | E_PARSE | E_ERROR | E_CORE_ERROR);
         }
 
+        if (interface_exists('Symfony\Bridge\PsrHttpMessage\HttpMessageFactoryInterface')) {
```

Comment:
In that case, we should have a configure option for all resolvers imo (so one config option in SecurityBundle too). I'm not sure I like that.